client/solve.go

```diff
@@ -412,15 +412,15 @@ func parseCacheOptions(opt SolveOpt) (*cacheOptions, error) {
 			}
 			cs, err := contentlocal.NewStore(csDir)
 			if err != nil {
-				return nil, err
+				logrus.Warning("local cache import at " + csDir + " not found due to err: " + err.Error())
```

Comment:
I think Tonis suggested statting the dir and checking `IsNotExist`?

Reply:
I think this is fine as well.